test/options.js

```diff
@@ -22,8 +22,15 @@ describe('options', () => {
     it('should work with negation patterns', () => {
       assert(isMatch('./x/y.js', '*.js', { matchBase: true }));
       assert(!isMatch('./x/y.js', '!*.js', { matchBase: true }));
-      assert(isMatch('./x/y.js', '**/*.js', { matchBase: true }));
```

Comment:
Not preserving the `./` is intentional—this is not a match according to minimatch.